docs/content/src/observability/logstash-kibana.md

```diff
@@ -1,5 +1,14 @@
 # Logging using Logstash and Kibana
 
+Ballerina distributed logging and analysis is supported by Elastic Stack.
```

Comment:
Let's add the link to Elastic Stack here and remove it from line 12.

Reply:
Fixed